lib/cdo/poste.rb

```diff
@@ -213,4 +209,25 @@ def self.send_message(message_name, recipient, params)
     })
   end
 
+
+  class DeliveryMethod
+
+    def initialize(settings)
+    end
+
+    def deliver!(mail)
+      content_type = mail.header['Content-Type'].to_s
+      raise ArgumentError, "Unsupported message type: #{content_type}" unless content_type =~ /^text\/html;/ && content_type =~ /charset=UTF-8/
+      sender = mail.from.first
+      raise ArgumentError, "Unsupported sender: #{sender}" unless sender == 'noreply@code.org'
+
+      subject = mail.subject.to_s
+      body = mail.body.to_s
+
+      recipient = Poste2::ensure_recipient(mail.to.first, ip_address:'127.0.0.1')
```

Comment:
what is this ip address used for? any reason to use the request's IP instead of localhost?

We (sometimes) have name here from the dashboard user, is that useful?